src/classes/BDI_FieldMappingCustomMetadata.cls

```diff
@@ -49,11 +70,65 @@ public with sharing class BDI_FieldMappingCustomMetadata implements BDI_FieldMap
     * TODO: Implement this method
     */
 
-    public Map<String,String> getFieldMap(String dataImportObjectName, 
-        String targetObjectName, List<String> dataImportFields) {
+    public Map<String,String> getFieldMap(String dataImportObjectName, String targetObjectName, List<String> dataImportFields) {
+
+        Map<String,String> dataImportFieldToTargetField = new Map<String,String>();
 
-        // TODO: Map fields based on CMT here
+        String ns = UTIL_Namespace.getNamespace();
+        //Retreive the Data Import Settings to determine the default field mapping set.
+        Data_Import_Settings__c dis = UTIL_CustomSettingsFacade.getDataImportSettings();
 
-        return new Map<String,String>();
+        String fieldMappingSetName = dis.Default_Data_Import_Field_Mapping_Set__c;
+        Data_Import_Field_Mapping_Set__mdt diFieldMappingSet;
+        Data_Import_Field_Mapping__mdt[]  diFieldMappings;
+        Data_Import_Object_Mapping__mdt diObjectMapping;
+
+        if(fieldMappingSetName != null){
```

Comment:
We've been aligning to putting a space between the if and the parens, and a space between closing paren and the curly brace - so "if (filedMappingSetName != null) {"